### PR TITLE
Adds WSO2 API Gateway to arch diagram

### PR DESCRIPTION
**Pull request by conbrad**
# Test Links:
[Percentile Calculator](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Fire Behaviour Advisory](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/fire-behaviour-advisory)
[HFI Calculator](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/hfi-calculator)
[FWI Calculator](https://wps-pr-2154.apps.silver.devops.gov.bc.ca/fwi-calculator)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ graph LR
 
     wf1["WFWX Fire Weather API</br>[Software System]"]
 
+    wso2["WSO2 API Gateway"]
```
Comment (Sybrand) [inline README.md:51]:
```suggestion
    ws02["WS02 API Gateway"]
```

File: README.md [hunk 2]
```diff
@@ -80,8 +82,9 @@ graph LR
     API-.->|"Read</br>[psycopg]"|Database
     API-.->|"Read</br>[JSON/HTTPS]"|CFFDRS_API
     API-.->|"Uses</br>[Reads from disk]"|Files
-    API-. "Fetch fire weather data</br>[JSON/HTTPS]" .->wf1
+    API-. "Fetch fire weather data</br>[JSON/HTTPS]" .->wso2
```
Comment (Sybrand) [inline README.md:85]:
```suggestion
    API-. "Fetch fire weather data</br>[JSON/HTTPS]" .->ws02
```

**Review by Sybrand — CHANGES_REQUESTED** [review]
I think it's WS02, not WSO2 ; I find lot's of references to it in confluence, but very little useful information.

File: README.md (hunk 1 above)
Comment (conbrad) [reply README.md:51]: Vendor says it's "O": https://wso2.com/api-manager/

File: README.md (hunk 2 above)
Comment (conbrad) [reply README.md:85]: Vendor says it's "O": https://wso2.com/api-manager/

**Review by Sybrand — APPROVED** [review]

**Pull request merged**